num_agents: 3
world_frame: "world"
odom_frame: "odom_fake"
agent_frame: "agent"
publish_pcl_world: false

agent_0:
  namespace: "firefly_0"
  position: [0.0, -5.0, 0.0]
  yaw: 0.0 # deg
  random_walk_mean_pos: 0.0
  random_walk_cov_pos: 0.0 
  random_walk_mean_ang: 0.0
  random_walk_cov_ang: 0.0

agent_1:
  namespace: "firefly_1"
  position: [0.0, 5.0, 0.0]
  yaw: 0.0 # deg
  random_walk_mean_pos: 0.0
  random_walk_cov_pos: 0.0
  random_walk_mean_ang: 0.0
  random_walk_cov_ang: 0.0

agent_2:
  namespace: "firefly_2"
  position: [0.0, -7.0, 0.0]
  yaw: 0.0 # deg
  random_walk_mean_pos: 0.0
  random_walk_cov_pos: 0.0
  random_walk_mean_ang: 0.0
  random_walk_cov_ang: 0.0
